include/wx/qt/private/winevent.h

```diff
@@ -458,4 +467,23 @@ class wxQtEventSignalHandler : public Widget, public wxQtSignalHandler< Handler
     }
 };
 
+// Used by wxTextEntry derived classes to toggle the dialog's default button
+// off and on. see src/qt/textentry.cpp for the reason.
+#define wxHANDLE_DIALOG_DEFAULT_BUTTON(Widget)              \
```

Comment:
This doesn't seem to be used at all any more, shouldn't it be removed?

Reply:
Oops! removed now, thanks.